curriculum/challenges/portuguese/02-javascript-algorithms-and-data-structures/es6/explore-differences-between-the-var-and-let-keywords.portuguese.md

```diff
@@ -3,26 +3,58 @@ id: 587d7b87367417b2b2512b3f
 title: Explore Differences Between the var and let Keywords
 challengeType: 1
 videoUrl: ''
-localeTitle: Explore as diferenças entre o var e let
+localeTitle: Explore as diferenças entre as palavras-chave var e let
 ---
 
 ## Description
-<section id="description"> Um dos maiores problemas com a declaração de variáveis ​​com a palavra-chave <code>var</code> é que você pode sobrescrever declarações de variáveis ​​sem um erro. <blockquote> var camper = &#39;James&#39;; <br> var camper = &#39;David&#39;; <br> console.log (campista); <br> // loga &#39;David&#39; </blockquote> Como você pode ver no código acima, a variável <code>camper</code> é originalmente declarada como <code>James</code> e, em seguida, sobrescrita como sendo <code>David</code> . Em um aplicativo pequeno, você pode não encontrar esse tipo de problema, mas quando o código se torna maior, você pode acidentalmente sobrescrever uma variável que não pretende sobrescrever. Como esse comportamento não gera um erro, a pesquisa e a correção de erros se tornam mais difíceis. <br> Uma nova palavra-chave chamada <code>let</code> foi introduzida no ES6 para resolver esse possível problema com a palavra-chave <code>var</code> . Se você fosse substituir <code>var</code> com <code>let</code> nas declarações de variáveis ​​do código acima, o resultado seria um erro. <blockquote> let campista = &#39;James&#39;; <br> let campista = &#39;David&#39;; // lança um erro </blockquote> Este erro pode ser visto no console do seu navegador. Portanto, ao contrário de <code>var</code> , ao usar <code>let</code> , uma variável com o mesmo nome só pode ser declarada uma vez. Observe o <code>&quot;use strict&quot;</code> . Isso habilita o Modo Estrito, que detecta erros comuns de codificação e ações &quot;inseguras&quot;. Por exemplo: <blockquote> &quot;use strict&quot;; <br> x = 3,14; // lança um erro porque x não está declarado </blockquote></section>
+<section id='description'>
+Um dos maiores problemas com a declaração de variáveis com a palavra-chave <code>var</code> é que você pode sobrescrever declarações de variáveis sem um erro.
+
+```js
+var camper = 'James';
+var camper = 'David';
+console.log(camper);
+// retorna 'David'
+```
+
+Como você pode ver no código acima, a variável `camper` é declarada originalmente como `James` e, em seguida, sobrescrita como sendo `David`.
+Em um aplicativo pequeno, você pode não encontrar esse tipo de problema, mas quando seu código se torna maior, você pode sobrescrever acidentalmente uma variável que não pretendia sobrescrever.
+Como esse comportamento não gera um erro, a pesquisa e a correção de erros se tornam mais difíceis. Because this behavior does not throw an error, searching and fixing bugs becomes more difficult.<br>
```

Comment:
There is English text in this paragraph.

`Because this behavior does not throw an error, searching and fixing bugs becomes more difficult.` 

This was already translated before in the portuguese text: `Como esse comportamento não gera um erro, a pesquisa e a correção de erros se tornam mais difíceis.`

So this English phrase could be removed